css/_welcome_page.scss

```diff
@@ -172,6 +173,31 @@ body.welcome-page {
         vertical-align: middle;
         line-height: $welcomePageButtonLineHeight;
         cursor: pointer;
+
+        &-go {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            min-height: $welcomePageGoButtonMinHeight;
+            font-size: $welcomePageGoButtonFontSize;
+            font-weight: $welcomePageGoButtonFontWeight;
+            border-radius: $welcomePageGoButtonBorderRadius;
+            padding: 0 7px;
+
+            // Unset common properties and values
+            // TODO: Specify common declarations inside .welcome-page-button to cancel unset
+            width: unset;
+            min-width: unset;
+            height: unset;
+            text-align: unset;
+            vertical-align: unset;
+            line-height: unset;
```

Comment:
I am sure these `unset`s are not elegant, yet required to leave `welcome-page-button` untouched on WelcomePage.web.js 

It would be appreciated if you would consider this as a first step of style normalization for the buttons on the welcome page :-)